### PR TITLE
Update cloudwatch alarms for heartbeat

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Add cloudwatch alarms for the new heartbeat api

### CODE REVIEW

File: aws/heartbeat/cloudwatch_logs.tf [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    principals {
+      type        = "Service"
+      identifiers = ["apigateway.amazonaws.com"]
+    }
+  }
+}
+
+resource "aws_cloudwatch_log_metric_filter" "heartbeat-500-errors-api" {
+  name           = "heartbeat-500-errors-api"
+  pattern        = "\"\\\" 500 \""
```
Comment (jzbahrai) [inline aws/heartbeat/cloudwatch_logs.tf:45]: @sastels based on your PR does this need to be [500]

Comment (sastels) [reply aws/heartbeat/cloudwatch_logs.tf:45]: Possibly just `[ERROR]` actually. ie to catch if the heartbeat crashes for some reason.

Comment (jimleroyer) [reply aws/heartbeat/cloudwatch_logs.tf:45]: I raised the question with Jumana. @jzbahrai Did you look in the  logs with 500s what will be the error format? To make sure the failed heartbeat will really go through the CW metric filter?

I know that @sastels had to change the one in API lambda hence I was wondering.

**Review by jimleroyer — COMMENTED** [review]
This is great overall! good job 💪 

I just wonder if the logs output for the heartbeat lambda will really go through the CW metric filter. I'll wait for Steve's take on it.

**Review by sastels — APPROVED** [review]
I think this will work! 😅

**Pull request merged**